### PR TITLE
Removed EOF requirement.

### PR DESCRIPTION
**Pull request by enzanki-ars**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -66,20 +66,11 @@ func main() {
...
-	for scanner.Scan() {
-		if scanner.Text() != "EOF" {
-			// Add new lines coming in to the output string
-			t += fmt.Sprintf("%v\n", scanner.Text())
-		} else {
-			// If a line comes in that is just `EOF` send the text and exit
-			postStdinToSlack(t)
-			break
-		}
+	bytes, err := ioutil.ReadAll(os.Stdin)
```
Comment (jackzampolin) [inline main.go:69]: What is the behavior here? Does each line trigger a new slack post or only `EOF` when what you are piping into ends?

**Comment by jackzampolin** [comment]
If this retains the current behavior LTGM

File: main.go (hunk 1 above)
Comment (enzanki-ars) [inline main.go:69]: My change waits for the pipe to end, then send a post with the piped text.

**Comment by enzanki-ars** [comment]
~~Looking back over the code, I see that this does cause the behavior to change. I might look into a flag to retain the current behavior.~~ (See below)

Sidenote, what does `LTGM` mean? I have not seen that before and Google is not helping me here...

File: main.go [hunk 2]
```diff
@@ -66,20 +66,11 @@ func main() {
...
-
-	// Start scanning through input
-	for scanner.Scan() {
-		if scanner.Text() != "EOF" {
-			// Add new lines coming in to the output string
-			t += fmt.Sprintf("%v\n", scanner.Text())
-		} else {
-			// If a line comes in that is just `EOF` send the text and exit
-			postStdinToSlack(t)
-			break
```
Comment (enzanki-ars) [inline main.go:82]: Correction: This code will not cause the behavior to change. Due to the lines 81 and 82, if EOF is read, the text before that will be sent and the program will stop, ignoring anything after it.

**Comment by enzanki-ars** [comment]
~~I added some more commits that fixed the eventual merge conflict, but GitHub seems to be [having issues](https://status.github.com/messages) displaying them right now.~~ (Fixed)

**Pull request merged**